edelphi/src/main/webapp/_scripts/gui/panel_admin/inviteactionsblockcontroller.js

```diff
@@ -66,6 +66,93 @@ InviteActionsBlockController = Class.create(BlockController, {
     }
     this._autoCompleteList.setVisible(this._autoCompleteList.getSize() > 0);
   },
+  
+  _getResponseError: function (response) {
+    if (response && response.messages && response.messages.length) {
+      for (var i = 0; i < response.messages.length; i++) {
+        if (response.messages[i].severity !== "OK") {
+          return response.messages[i].message;
+        }
+      }
+    }
+    
+    return null;
+  },
+  
+  _createInvitationRequest: function (invitationMessage, userId, queryId, email, progressCallback) {
+    return function (callback) {
+      JSONUtils.request(CONTEXTPATH + '/panel/admin/createinvitation.json', {
+        parameters: {
+          invitationMessage: invitationMessage,
+          email: email,
+          userId: userId,
+          queryId: queryId
+        },
+        onSuccess : function(response) {
+          var error = this._getResponseError(response);
+          progressCallback(error);
+          callback(null, {
+            error: error,
+            email: email
+          });
+        }.bind(this),
+        onFailure: function (response) {
+          var error = this._getResponseError(response);
+          progressCallback(error);
+          callback(null, {
+            error: error,
+            email: email
+          });
+        }.bind(this)
+      });  
+    }.bind(this)
+  },
+  
+  _showMessage: function (severity, message) {
+    var eventQueue = getGlobalEventQueue();
+    
+    switch (severity) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Add a "default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)